browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsSignupDialog.java

```diff
@@ -116,10 +116,12 @@ public void onClick(DialogInterface dialog, int which) {
                 OnboardingPrefManager.getInstance().setOnboardingNotificationShown(false);
 
                 neverShowOnboardingDialogAgain();
-
-                // Enable ads
-                if (BraveActivity.getBraveActivity() != null) {
-                    BraveActivity.getBraveActivity().openRewardsPanel();
+                try {
+                    // Enable ads
+                    if (BraveActivity.getBraveActivity() != null) {
+                        BraveActivity.getBraveActivity().openRewardsPanel();
+                    }
+                } catch (Exception e) { // sujit
```

Comment:
Should be ActivityNotFoundException. Also not sure what this comment implies.

Reply:
updated